packages/scheduling/src/managers/scheduler.manager.ts

```diff
@@ -5,10 +5,21 @@ import {SchedulingModuleKeyname} from "../scheduling.module.keyname";
 import {SchedulerInterface} from "../interfaces/scheduler.interface";
 import {LogHandlerInterface} from "@pristine-ts/logging";
 
+/**
+ * The Scheduler Manager runs the scheduled tasks.
```

Comment:
We should add that the scheduled task cannot specify the interval that they want to be triggered. It's only a mechanism to understand that must be plugged in into a CRON or something. We should also refer them to the event bridge module and how we understand it's a recurring timer and we use this to trigger the scheduler manager.

We should also specify that the scheduled task will be triggered at a recurring interval and that its their own responsibility to implement a logic of executing or not based on how long it's been since the last execution.